src/main/java/co/org/osso/api/BookService.java

```diff
@@ -0,0 +1,52 @@
+package co.org.osso.api;
+
+import org.springframework.stereotype.Service;
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class BookService {
+
+    Map<String, Book> mapBooks = new HashMap<>();
+    int id;
+
+    public Book createBook(Book book) {
+        book.setBookID(""+id++);
+        mapBooks.put(book.getBookID(), book);
+        return book;
+    }
+
+    public Book getBook(String id) {
+        if (mapBooks.get(id) != null){
+            return mapBooks.get(id);
+        } else {
+            return new Book();
+        }
+    }
+}
+
+class Book{
+    String bookID;
```

Comment:
Let's change this data type to be an `Integer`. (not `int`, because we want it to be `null` for the POST request.

Reply:
Done